platform-sdk/swirlds-common/src/main/java/com/swirlds/common/wiring/builders/TaskSchedulerBuilder.java

```diff
@@ -300,7 +301,7 @@ private FractionalTimer buildBusyTimer() {
         if (metricsBuilder == null || !metricsBuilder.isBusyFractionMetricEnabled()) {
             return NoOpFractionalTimer.getInstance();
         }
-        if (concurrent) {
+        if (type == TaskSchedulerType.CONCURRENT) {
             throw new IllegalStateException("Busy fraction metric is not compatible with concurrent wires");
```

Comment:
not compatible with concurrent ~~wires~~ *schedulers*

Reply:
fixed